applications/FluidDynamicsApplication/custom_constitutive/bingham_2d_law.cpp

```diff
@@ -0,0 +1,158 @@
+//    |  /           |
```

Comment:
Same comment, as it is a new addition I'd properly format the file and comments for the sake of readability.

Reply:
Besides, I'd move the "one line" implementations in the source file to the header.